src/sidebar/components/test/tag-editor-test.js

```diff
@@ -61,6 +61,10 @@ describe('TagEditor', function() {
   function selectOptionViaDelimiter(wrapper) {
     wrapper.find('input').simulate('keydown', { key: ',' });
   }
+  // Simulates typing ","
+  function selectOptionViaTab(wrapper) {
+    wrapper.find('input').simulate('keydown', { key: 'Tab' });
+  }
   // Simulates typing the down arrow
```

Comment:
I don't think this comment is necessary here (I concede this comment predates these changes...).

Reply:
I think you're right, i'll refactor